tests/test_neighborhood.py

```diff
@@ -201,13 +205,12 @@ def test_calculate_new_coordinate_larger_magnitude(self):
         nc = NeighborhoodConfig(dims, radius=2, min_initialized=3)
 
         n = Neighborhood(nc, cd, Coordinate([0, 0]))
-        magnitude = 3
+        magnitude = 20
+        new_coord = n.calculate_new_coordinate(magnitude)
 
         # Run it multiple times to make sure no values are changing
-        self.assertEqual(Coordinate([2, 2]),
-                         n.calculate_new_coordinate(magnitude))
-        self.assertEqual(Coordinate([2, 2]),
-                         n.calculate_new_coordinate(magnitude))
+        self.assertEqual(new_coord, Coordinate([2, 2]))
+        self.assertEqual(new_coord, Coordinate([2, 2]))
```

Comment:
If we are not re-running `calculate_new_coordinate`, then I think we can remove this line.

Reply:
good catch. Fixed